templates/tfengine/recipes/project.hcl

```diff
@@ -84,7 +84,7 @@ schema = {
             properties = {
               api = {
                 type = "string"
-                description = "The API to apply the roles to. Example: healthcare.googleapis.com"
+                description = "The API whose default Service Agent will be force-created and granted the role. Example: healthcare.googleapis.com."
```

Comment:
nit: s/role/roles

Reply:
DId you mean `... granted the role/roles`?

Reply:
`granted the roles.`

with the additional s because I noticed the roles field is a list.

We usually use `s/before/after` as a shorthand for replacement. It's from sed command syntax. Sorry for the confusion.

Reply:
Ohh got it. Sure, np! Done